app/models/ability.rb

```diff
@@ -240,20 +232,20 @@ def signed_in_with_volunteers_coordinator_role(user)
     # ids of all the conferences for which the user has the 'volunteers_coordinator' role
     conf_ids_for_volunteers_coordinator = []
     conf_ids_for_volunteers_coordinator =
-        Conference.with_role(:volunteer_coordinator, user).pluck(:id) if user.has_role? :volunteer_coordinator, :any
+        Conference.with_role(:volunteers_coordinator, user).pluck(:id) if user.has_role? :volunteers_coordinator, :any
```

Comment:
There is not a reason any more, no. Now we already check that in https://github.com/nishanthvijayan/osem/blob/fix-roles-authorization/app/models/ability.rb#L100

It is important to check somewhere so that we can reduce the code executed for authorization. We only need to execute the relative lines, not everything for everyone. As we are still having a lot of calculations being made for authorization.